medcat/config_rel_cat.py

```diff
@@ -129,3 +182,21 @@ class ConfigRelCAT(MixingConfig, BaseModel):
     class Config:
         extra = 'allow'
         validate_assignment = True
+
+    @classmethod
+    def load(cls, load_path: str = "./") -> "ConfigRelCAT":

```

Comment:
This method seems to work differently from the other `<Some>Config.load` method. As in, it doesn't load from the `load_path` directly, but rather from `<load_path>/.config.json`. Which is why the test I've added before now fails (`tests.utils.test_config_utils.RelCATConfigTests.test_can_load_old_format_correctly`).